elastic-apm-agent-java8/src/main/java/apm/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/**
+ * This project contains the Java 8 build of the elastic apm java agent.
+ */
+package apm;
```

Comment:
[question] why not use a more "traditional" package like `co.elastic.apm.java8` so it does not conflicts with the other `JavadocPlaceHolder.java` and still fits a bit the other packages even if we don't really publish it ?

Reply:
I decided to go with `co.elastic.apm.premain.java7/java8` to be a subpackage of the other non-shaded premain classes.